examples/molecular_dynamics/example_molecular_dynamics.cpp

```diff
@@ -16,28 +16,6 @@
 #include <iostream>
 #include <type_traits>
 
-template <class MemorySpace>
-struct Neighbors
-{
-  Kokkos::View<ArborX::Point *, MemorySpace> _particles;
-  float _radius;
-};
-
-template <class MemorySpace>
-struct ArborX::AccessTraits<Neighbors<MemorySpace>, ArborX::PredicatesTag>
-{
-  using memory_space = MemorySpace;
-  using size_type = std::size_t;
-  static KOKKOS_FUNCTION size_type size(Neighbors<MemorySpace> const &x)
-  {
-    return x._particles.extent(0);
-  }
-  static KOKKOS_FUNCTION auto get(Neighbors<MemorySpace> const &x, size_type i)
-  {
-    return attach(intersects(Sphere{x._particles(i), x._radius}), (int)i);
-  }
-};
```

Comment:
I am wondering if showcasing the traits is not a bad idea here in case users want to plug in their own particle data structure that has the position amongst other things.
Thoughts?

Reply:
I do think we should have at least one example where we demonstrate attaching. I did not think about a user wanting to attach to their own particles data structure. Could be useful.